### PR TITLE
Show backup status of select system apps

### PR DESCRIPTION
**Pull request by grote**
A nice improvement before we tackle #117.

device-2020-10-23-135659

### CODE REVIEW

**Review by stevesoltys — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -91,6 +91,10 @@
 
     <!-- App Backup and Restore State -->
 
+    <string name="backup_sms">SMS text messages</string>
```
Comment (chirayudesai) [inline app/src/main/res/values/strings.xml:94]: "SMS text messages" one feels somewhat redundant - and this might include MMS too.

**Review by chirayudesai — COMMENTED** [review]
@mikeNG string review appreciated.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (grote) [reply app/src/main/res/values/strings.xml:94]: Google uses the same language though: https://github.com/stevesoltys/seedvault/issues/117#issuecomment-701335659
Also, I think MMS might not always get included as Google's non-free backup as a special class for that.

**Comment by mikeNG** [comment]
> @mikeNG string review appreciated.

Looks fine

**Pull request merged**